hunting-buddy.lic

```diff
@@ -81,6 +77,7 @@ class HuntingBuddy
       stop_on_low_skills = info['stop_on_low']
       stop_on_boxes = info['boxes']
       stop_on_no_moons = info['moons']
+      stop_on_encumbrance = info['encumbrance']
```

Comment:
I see that you followed the pattern here for boxes/moons/etc, but it feels like those property names aren't super clear what they do to the average person.

Should we try to update the hunt config to something like:
```
  stop_on:
    low_skills:
    - Theurgy
    encumbrance: true
    max_boxes: false
 ```
 etc.?

Reply:
Yeah, I had half a mind to add support for `stop_on_boxes` and `stop_on_moons` so it'd be clearer but then decided against it because of existing player base uses the ambiguous names already. Not an excuse for the new encumbrance setting to be that way, but as you noticed, I just went with the convention that had been set